src/sonic-yang-models/yang-models/sonic-vrf.yang

```diff
@@ -0,0 +1,49 @@
+module sonic-vrf {
+	namespace "http://github.com/Azure/sonic-vrf";
+	prefix vrf;
+	
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC VRF";
+
+	revision 2021-03-30 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-vrf {
+		container VRF {
+			description "Vrf configuration.";			 
+
+			list VRF_LIST {
+				key "vrf_name";
+
+				leaf vrf_name {
+					type string {
+						pattern "default|Vrf[a-zA-Z0-9_-]+" {
```

Comment:
As discussed in the meeting please add a comment and reference how vrf with name `default` is handled in swss.

Reply:
Explicitly checking for "default" value and ignoring/handling of the same is not done in vrfmgr.cpp (in sonic-swss//cfgmgr/vrfmgr.cpp). Hence removing the "default" keyword from the pattern and in the sample config db json file.